torch/_dynamo/variables/functions.py

```diff
@@ -113,7 +113,7 @@ def __init__(self, fn, is_constant=False, **kwargs):
             self.is_constant = False
 
         assert isinstance(
-            fn, (types.FunctionType, torch.jit.ScriptFunction)
+            fn, (types.MethodType, types.FunctionType, torch.jit.ScriptFunction)
```

Comment:
Shouldn't this be handled by UserMethodVariable?

Reply:
This is probably due to the issue i stumbled onto where promoting something to UMV opened up more inlining opportunities and introduced many graph breaks to models that currently didn't graph break.
https://github.com/pytorch/pytorch/blob/master/torch/_dynamo/variables/builder.py#LL519C17-L519C17

IMO we should make a point of weeding out this kind of thing even if it means regressing (or disabling) some models and then hopefully fixing them back a better way.